Rakefile

```diff
@@ -37,66 +37,72 @@ Rake::TaskManager.record_task_metadata = true
 namespace :setup do
   desc 'Install all the prerequisites on you system'
   task :prerequisites do
-    msg "Install rubygem dependencies locally"
-    system "bundle install"
+    msg 'Install rubygem dependencies locally'
+    system 'bundle install'
 
-    msg "Download and vendor the necessary cookbooks locally"
-    system "bundle exec berks vendor cookbooks"
+    msg 'Download and vendor the necessary cookbooks locally'
+    system 'bundle exec berks vendor cookbooks'
 
     msg "Current chef environment => #{ENV['CHEF_ENV_FILE']}"
-    if File.exist?(ENV['CHEF_ENV_FILE'])
-      puts "You need to configure an Environment under 'environments/'. Check the README.md".red
-      puts "If you just have a different chef environment name run:"
-      puts "  # export CHEF_ENV=#{"my_new_environment".yellow}"
+    unless File.exist?(ENV['CHEF_ENV_FILE'])
+      puts 'You need to configure an Environment under \'environments/\'. Check the README.md'.red
+      puts 'If you just have a different chef environment name run:'
+      puts "  # export CHEF_ENV=#{'my_new_environment'.yellow}"
     end
   end
 
   desc 'Setup the Chef Delivery Cluster that includes: [ Chef Server | Delivery Server | Build Nodes ]'
-  task :cluster => [:prerequisites] do
-    msg "Setup the Chef Delivery cluster"
+  task cluster: [:prerequisites] do
+    msg 'Setup the Chef Delivery cluster'
     chef_zero 'setup'
   end
 
   desc 'Setup a Chef Server'
   task :chef_server do
-    msg "Create a Chef Server"
+    msg 'Create a Chef Server'
     chef_zero 'setup_chef_server'
   end
 
   desc 'Create a Delivery Server & Build Nodes'
   task :delivery do
-    msg "Create Delivery Server and Build Nodes"
+    msg 'Create Delivery Server and Build Nodes'
     chef_zero 'setup_delivery'
   end
 
   desc 'Activate Analytics Server'
-  task :analytics => [:chef_server] do
-    msg "Setup Chef Analytics so we can see what is going on in our cluster"
+  task analytics: [:chef_server] do
+    msg 'Setup Chef Analytics so we can see what is going on in our cluster'
```

Comment:
I like the switch in syntax. But as my overall comment says I wish it were its own pr.